dataset/dictionary/weapons.json5

```
@@ -1462,7 +1462,7 @@
     en: "Jadefall's Splendor",
     ja: "碧落の瓏",
     zhCN: "碧落之珑",
-    // pronunciationJa: "へきらくのろう", // TODO Need Check
+    // pronunciationJa: "へきらくのろう",
```

Please revert this if there are no official sources.

```suggestion
    // pronunciationJa: "へきらくのろう", // TODO Need Check
```